src/assets/Standards/CollectibleStandards/ERC1155/ERC1155Standard.test.ts

```diff
@@ -1,9 +1,8 @@
 import Web3 from 'web3';
-import HttpProvider from 'ethjs-provider-http';
 import nock from 'nock';
 import { ERC1155Standard } from './ERC1155Standard';
 
-const MAINNET_PROVIDER = new HttpProvider(
+const MAINNET_PROVIDER = new Web3.providers.HttpProvider(
```

Comment:
Changed because the `Web3` constructor takes an argument whose type does not match the (new) type that `ethjs-provider-http` returns. I didn't see a real reason to mix and match libraries here when this class seems to take an instance of Web3.